static/common/js/registration.js

```diff
@@ -0,0 +1,47 @@
+getById = id => document.getElementById(id);
+
+
+let nick = getById("nickname");
+let firstName = getById("first_name");
+let lastName = getById("last_name");
+let age = getById("age");
+let sex = getById("sex");
+let gdpr = getById("gdpr");
+
+getById("after_signature").style.opacity = 0;
+firstName.addEventListener('keyup', () => getById("text_fname").innerHTML = firstName.value);
+lastName.addEventListener('keyup', () => getById("text_lname").innerHTML = lastName.value);
+age.addEventListener('keyup', () => getById("text_age").innerHTML = age.value);
+age.addEventListener('change', () => getById("text_age").innerHTML = age.value);
+
+nick.addEventListener('keyup', () => {
+    let before = ".character::before{content: '" + nick.value + ": '}";
+    let head = document.head.innerHTML;
+    document.head.innerHTML = head + "<style>" + before + "</style>";
+    document.getElementById("signature").innerHTML = nick.value;
+});
+
+sex.addEventListener('change', () => {
+    let sex_suffix = sex.value == 1 ? "a" : "";
+    let spans = document.querySelectorAll('.sex_suffix');
+
+    let motive = "Co tě dovedlo k rozhodnutí zaregistrovat se na náš web? Je nějaký speciální důvod, proč bys rád" + sex_suffix + " patřil" + sex_suffix + " mezi nás?";
```

Comment:
It would be neat if this would get extracted from the placeholder on load, not duplicated

Reply:
Didn't think of a way how to do that, you have an idea how to localize the place for suffix?

Reply:
The select doesn't work?